### ISSUE TITLE
Switching between PR tabs adds duplicate job info

### ISSUE THREAD

**Issue by TimvdLippe** [dominique-mueller/web-extension-github-travis-status#29]
Take for example https://github.com/telegraf/telegraf/pull/415 There should be 3 sub-jobs on that PR. When you click any other tab (such as "files") and then go back, you should still see 3 sub-jobs. However, switching yet another time now shows 6 jobs. This increases by 3 for every tab-change.

![image](https://user-images.githubusercontent.com/5948271/39942346-84f754c2-555f-11e8-82dc-e8dd7fff9017.png)

**Comment by itsdevdom** [comment]
Yep, can be reproduced. Might be an issue with the regex at https://github.com/dominique-mueller/web-extension-github-travis-status/blob/develop/src/background/background.ts#L65.
